modules/ROOT/pages/administration/databases.adoc

```diff
@@ -903,6 +926,119 @@ ALTER DATABASE nonExisting IF EXISTS SET TOPOLOGY 1 PRIMARY 0 SECONDARY
 [role="statsonlyqueryresult"]
 0 rows
 
+[role=enterprise-edition]
+[[administration-databases-alter-database-options]]
+=== Options
+
+////
+[source, cypher, role=test-setup]
+----
+CREATE DATABASE `options-example`;
+----
+////
+
+The `ALTER DATABASE` command can be used to set or remove specific options, which are listed below.
+This is a subset of the options allowed for `CREATE DATABASE` as the remaining create options are only applied during the creation of the database.
+
+[options="header"]
+|===
+
+| Key | Value | Description
+
+| `txLogEnrichment`
+| `FULL` \| `DIFF` \| `NONE`
+|
+Defines level of enrichment applied to transaction logs for Change Data Capture (CDC) purposes.
+For details about enrichment mode, see link:{neo4j-docs-base-uri}/cdc/{page-version}/getting-started/enrichment-mode[Change Data Capture Manual -> Getting Started -> Enrichment mode].
+
+|===
+
+[NOTE]
+====
+There are no available `OPTIONS` values for composite databases.
+====
+
+.+ALTER DATABASE SET OPTION+
+======
+
+// skip test until we make the change_data_capture feature flag on by default.
+
+.Query
+[source, cypher, test-skip]
```

Comment:
```suggestion
[source, cypher, role="test-skip"]
```
Right now it does work even if you don't put it within the `role` attribute, but I don't want to guarantee that it always will, so please apply this change to further examples below as well.